src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -58,11 +56,12 @@
  */
 public final class Office365ConnectorWebhookNotifier {
 
-    private static final Gson gson = new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.IDENTITY).create();
+    private static final Gson gson = new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.IDENTITY)
+            .setPrettyPrinting().create();
```

Comment:
`setPrettyPrinting` converts JSON file from single line to something what is seen below in `started.json` file - much easier to debug and compare